Snowdog_Components/components/Templates/contact-us/_contact-us.scss

```diff
@@ -19,12 +19,15 @@
         @include visually-hidden();
     }
 
-    &__description,
-    &__description p {
+    &__description {
         margin: $contact-us__description-margin;
         color: $contact-us__description-color;
     }
 
+    &__description p {
+        color: $contact-us__description-color;
+    }
+
     &__button {
```

Comment:
For me this is ok but perhaps we should move things the other way. Margin just for description and margin, color leave as it was. 

```
    &__description {	
        margin: $contact-us__description-margin;
    }	 

    &__description,
    &_description p {
        color: $contact-us__description-color;
    }
```